packages/react-components/src/components/VideoTile.tsx

```diff
@@ -192,12 +192,14 @@ const videoTileMoreMenuProps = {
 /* @conditional-compile-remove(pinned-participants) */
 const VideoTileMoreOptionsButton = (props: { contextualMenu?: IContextualMenuProps }): JSX.Element => {
   const { contextualMenu } = props;
+  const ids = useIdentifiers();
   if (!contextualMenu) {
     return <></>;
   }
   return (
     <IconButton
       styles={moreButtonStyles}
+      data-ui-id={ids.videoTileMoreOptionsButton}
```

Comment:
nit: I think we are allowed to just assign a string directly like so:
```
    <IconButton
      data-ui-id="video-tile-more-options-button"
```
because the `_Identifiers` is internal. But this is fine too.